package.json

```diff
@@ -70,8 +70,10 @@
   "resolutions": {
     "graphql": "15.4.0",
     "@jahia/data-helper/graphql": "^15.4.0",
-    "trim-newlines": "3.0.1",
-    "css-what": "5.0.1"
+    "**/meow/trim-newlines": "3.0.1",
```

Comment:
we should also use caret versions in resolutions

Reply:
Sure I will update those ones